pom.xml

```diff
@@ -47,4 +48,12 @@
         <component.name>kafka</component.name>
         <io.confluent.kafka-images.version>7.4.1-0</io.confluent.kafka-images.version>
     </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
shouldn't this be inherited fro common-docker -> common -> common-parent? do we need it?

Reply:
In the common's pom.xml, `junit` is added under the `dependencyManagement` section instead of `dependencies` section, so this dependency is not directly inheritable, instead, Child projects must explicitly declare the dependency in their own `<dependencies>` section.  specifying the `groupId` and `artifactId`, but they can omit the version since it will be inherited from the parent's `<dependencyManagement>`.